server/config/persist_options.go

```diff
@@ -196,6 +197,21 @@ func (o *PersistOptions) SetMaxReplicas(replicas int) {
 	o.SetReplicationConfig(v)
 }
 
+// UseRaftV2 set some config for raft store v2 by default temporary.
+// todo: remove this after raft store support this.
+// disable merge check
+// disable split buckets
+func (o *PersistOptions) UseRaftV2() {
+	v := o.GetScheduleConfig().Clone()
+	if v.StoreConfigSynced {
+		return
+	}
+	v.MaxMergeRegionSize = 0
+	v.MaxMovableHotPeerSize = math.MaxInt64
+	v.StoreConfigSynced = true
```

Comment:
Will it become false after it becomes true?

Reply:
never become true, because the default config should be update only once. so the default config can be modified by manually.

Reply:
What if we disable v2?

Reply:
region merge and bucket split?

Reply:
I mean we enable it and then disable it.

Reply:
It will edit the merge size again after it restart.